judge/models/profile.py

```diff
@@ -243,15 +243,16 @@ def calculate_points(self, table=_pp_table):
         from judge.models import Problem
         public_problems = Problem.get_public_problems()
         data = (
-            public_problems.filter(submission__user=self, submission__points__isnull=False)
+            public_problems.filter(submission__user=self, submission__is_archived=False,
```

Comment:
We should also remove archived submissions from problem stats I think.

Reply:
done. i also added `is_archived=False` to all of the queries that i found.